java/app/src/test/java/studentdb/database/StudDatabaseTest.java

```diff
@@ -9,7 +9,7 @@
 import java.util.Optional;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import studentdb.model.Stud;
+
```

Comment:
```suggestion
import studentdb.model.Stud;
``` 
Don't mess up with the libraries! They are there for a reson. Test completed!